client/src/styles/entitlements/_overrides.scss

```diff
@@ -11,7 +11,12 @@ button.unavailable:hover, li.unavailable > a:hover {
   opacity: 0;
 }
 
-.navbar-brand .navbar-brand-icon, .modal-dialog .modal-footer > img {
-  height: 35px;
-}
+.about-modal-pf {
+  background-color: #1a1a1a;
+  background-image: url('/assets/img/About-Modal-background.png');
```

Comment:
We including this with RCUE down the road?

There's kind of a hidden aspect of the Create-React-App with Webpack... you can make the path relative, like so
```
background-image: url('../images/About-Modal-background.png');
```

And it should run and compile with the updated absolute path.

Reply:
I'm not sure what the RCUE plan is for this. It currently copies images from patternfly so it does not have the correct image for this.

Reply:
Excellent, all images in one place now.